verification/curator-service/ui/cypress/integration/components/BulkCaseForm.spec.ts

```diff
@@ -33,6 +33,7 @@ describe('Bulk upload form', function () {
         cy.wait('@upsertCase');
         cy.wait('@upsertCase');
         cy.contains('Success! Created 2 new rows.');
+        cy.get('[data-testid="progress"]').should('not.be.visible');
```

Comment:
Perhaps `not.exist` ?

Reply:
I've typically used that, but thought that `visible` was a better description of the user behavior we were testing, here.

I can switch to `exist` for now, since it's used elsewhere. I'll take a look at which might be better to use throughout.